cluster/saltbase/salt/kube-proxy/kube-proxy.manifest

```diff
@@ -75,6 +75,13 @@ spec:
     - -c
     - echo -998 > /proc/$$$/oom_score_adj && kube-proxy {{api_servers_with_port}} {{kubeconfig}} {{cluster_cidr}} --resource-container="" {{params}} 1>>/var/log/kube-proxy.log 2>&1
     {{container_env}}
+    livenessProbe:
```

Comment:
the current livenessProbe would probably not catch the previous issue as it is a k8s daemon that also binds /healthz...